### DISCUSSION TITLE
Allow arrow key navigation in more places

### DISCUSSION THREAD

**Discussion by revelsix** [ppy/osu#21674] [Ideas] [answered]
As of now, arrow keys are only really useful for navigating through song select. Meanwhile, keyboard navigation on the home menu consists of either just hitting enter to go straight to song select, or hitting seemingly random keys (to a new player) to navigate into certain components. 
Here are the ones I've noticed:
![image](https://user-images.githubusercontent.com/45189851/208198038-ef444a4c-a79b-4ced-a1de-86dfec1b1f30.png)
![image](https://user-images.githubusercontent.com/45189851/208198205-130b0e0c-6159-45e6-a79c-bf203f7984fd.png)
Using arrow keys to be able to move through these menus would make keyboard navigation much simpler in my opinion.

Another potential area where arrow keys could be useful is in the mod select screen. Right now, only 3 categories of mods are accessible via keyboard and the enter or escape keys are used to close the menu. Using arrows to hover over mods and the enter key to select seems like a better (or at least alternate) way to select mods via keyboard.

**Comment by peppy** [comment]
I'm not against this being added, for what it's worth.

**Comment by Walavouchey** [comment, marked as answer]
tracked in https://github.com/ppy/osu/issues/8048
